package.json

```diff
@@ -24,10 +24,9 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@polymer/polymer": "^3.0.0",
-    "@polymer/iron-jsonp-library": "^3.0.0-pre.19"
+    "@polymer/iron-jsonp-library": "^3.0.0-pre.18"
   },
   "devDependencies": {
-    "@polymer/iron-component-page": "^3.0.0-pre.19",
-    "@webcomponents/webcomponentsjs": "^2.0.0"
+    "@polymer/iron-component-page": "^3.0.0-pre.18"
   }
```

Comment:
Are the changes to package.json intentional?

Reply:
Hm, odd. Recommend reverting, it's nice for this to have a dev dep on webcomponentsjs

Reply:
Done.